src/envoy/http/authn/http_filter.h

```diff
@@ -18,15 +18,26 @@
 #include "authentication/v1alpha1/policy.pb.h"
 #include "common/common/logger.h"
 #include "server/config/network/http_connection_manager.h"
+#include "src/envoy/http/authn/context.pb.h"
 
 namespace Envoy {
 namespace Http {
 
+namespace iaapi = istio::authentication::v1alpha1;
```

Comment:
per c++ style guide,  this is not allowed in header file.

Reply:
Done